sage_modabvar/abvar.py

```diff
@@ -445,6 +447,36 @@ def label(self):
         """
         degen = str(self.degen_t()).replace(' ','')
         return '%s%s'%(self.newform_label(), degen)
+    
+    def newform(self, names=None):
+        """
+        Return the newform `f` such that this abelian variety is isogenous to
+        the newform abelian variety `A_f`. If this abelian variety is not
+        simple, raise a ValueError.
+
+        INPUT: "names" - if the newform has coefficients in a number field,
```

Comment:
The input and output blocks are formatted incorrectly.  Look it up in the Sage dev guide -- there is a template there.